internal/util/util.go

```diff
@@ -144,6 +145,21 @@ func GetSecretPassword(clientset kubernetes.Interface, db, suffix, Namespace str
 
 }
 
+// GetStandardImageTag takes the current image name and the image tag value
+// stored in the pgcluster CRD and, if the image being used is the
+// crunchy-postgres-gis-ha container with the corresponding tag, it returns
+// the tag without the addition of the GIS version. This tag value can then
+// be used when provisioning containers using the standard containers tag.
+func GetStandardImageTag(imageName, imageTag string) string {
+
+	if imageName == "crunchy-postgres-gis-ha" && strings.Count(imageTag, "-") > 2 {
+		re := regexp.MustCompile(`(.+-[\d|\.]+)-[\d|\.]+?(-[\d|\.]+.*)`)
```

Comment:
If this is a `MustCompile`, we could keep this at the top of the file in a `var` and only initialize once

Reply:
That sounds good to me. Updated, with a more specific variable name. Thanks!